### DISCUSSION TITLE
Action capturing a string (I think) with color in it.

### DISCUSSION THREAD

**Discussion by elminster3k** [scandum/tintin#152] [Script help]
I have an action set up for something and when I view the variable, it has color in it (as I see it on the mud) which is unlike anything else I currently capture.  The variable it stores is visibly an integer, but I cant use it as an integer, which I guess is due to the color?  Is there a way to capture a string but strip out any color or just treat it as plain text?  I have set up so many actions without issue here, but this one is just odd for me to understand.  https://i.imgur.com/uB8n2vg.png   From the screen capture, if I try to use that fullness var as a number, it just is not working for me.

**Comment by scandum** [comment]
I assume you're capturing the value with a color trigger. Using:
```
#line strip #var fullness <val>
```
might do the trick.

**Reply by elminster3k** [reply]
First of all, Merry Christmas!  Thanks for replying.  Line looks like this:  https://i.imgur.com/Abe5dj1.png
Capturing like this:  `#act {~\{S\}~ %1} {#var slimeinfo %1} `  (this end up being a colored variable which is odd to me)
Then grabbing fullness like this:  #regex {$slimeinfo} {Fullness:%1%} {#var fullness &1}

I will try the line strip out.

**Comment by scandum** [comment]
If you start an #action with a `~` it turns into a color trigger. Easiest fix is likely:
```
#act {\~\{S\}~ %1} {#var slimeinfo %1}
```

And a merry Christmas to you too!  😄

**Reply by elminster3k** [reply]
Wow!  Man, you are a life saver.  Thanks so much!
